### ISSUE TITLE
After adding and playing 'floating' layout, Amethyst stop managing window on my Mac

### ISSUE THREAD

**Issue by jasonhaihuang** [ianyh/Amethyst#1081]
**Describe the bug**
After adding and playing 'floating' layout, Amethyst stop managing window on my Mac. I tried relaunch, delete and install (unzip it) again, the app is not organise the windows anymore. 

**Applications:**
All the applications are not be organised

**To Reproduce**
launch Amethyst, press option+shift+space to check if the windows are organised by different layout. Previously the windows will relayout according to the layout currently choosing, but nothing happened now while switching.

**Expected behavior**
Windows can be relayout as expected.

**Screenshots**

**Versions:**
 - macOS: 10.15.7 
 - Amethyst: 0.15.4 (91)
 
**Debug Info**
```
Version: 0.15.4 (91)

OS version: Version 10.15.7 (Build 19H114)

Has permissions: true

Screens:
	(0.0, 0.0, 3840.0, 2160.0) [(0.0, 0.0, 3840.0, 2160.0)]

Manageable applications:
	talagent (com.apple.talagent)
	Google Chrome (com.google.Chrome)
	Preview (com.apple.Preview)
	Code (com.microsoft.VSCode)
	Microsoft Outlook (com.microsoft.Outlook)
	Slack (com.tinyspeck.slackmacgap)
	Activity Monitor (com.apple.ActivityMonitor)
	Terminal (com.apple.Terminal)
	Calendar (com.apple.iCal)
	TextMate (com.macromates.TextMate)
	Sourcetree (com.torusknot.SourceTreeNotMAS)
	Finder (com.apple.finder)
	storeuid (com.apple.storeuid)

Configuration:
mouse-resizes-windows: 0
debug-layout-info: 1
mod1: (
    option,
    shift
)
window-minimum-height: 0
enables-layout-hud-on-space-change: 1
window-margins: 1
mod2: (
    option,
    shift,
    control
)
screen-padding-left: 0
enables-layout-hud: 1
screen-padding-bottom: 0
follow-space-thrown-windows: 1
floating: (
)
mouse-follows-focus: 0
mouse-swaps-windows: 0
screen-padding-top: 0
window-minimum-width: 0
restore-layouts-on-launch: 1
window-margin-size: 20
screen-padding-right: 10
window-resize-step: 2
ignore-menu-bar: 0
focus-follows-mouse: 0
floating-is-blacklist: 0
float-small-windows: 0
new-windows-to-main: 0
layouts: (
    tall,
    "tall-right",
    fullscreen,
    "middle-wide",
    column,
    bsp
)
use-canary-build: 0```

**Additional context**
No

**Comment by ianyh** [comment]
The floating layout disables all window tiling. If you want windows to be tiled do not use the floating layout.

**Comment by ianyh** [comment]
To clarify, switching to other layouts is still not tiling any windows?

**Comment by sebastiant** [comment]
Experiencing the same problem on macOS 10.15.6 and Amethyst 0.15.4 (91), after testing different settings for floating windows.
Switching between layouts shows the dialogue for the selected layout, but windows are not rearranged

**Comment by ianyh** [comment]
Oh, I think I see the problem. The floating option in your configuration is set to float everything by default. You want to set it to float applications listed rather than floating except applications listed.

**Comment by sebastiant** [comment]
@ianyh Correct! But simply changing the configuration and disabling and enabling Amethyst wasn't enough. Got it to work now by restarting the application. Thanks for you help

**Comment by jasonhaihuang** [comment]
Thanks a lot!

**Comment by maikeldaloo** [comment]
Hello @ianyh, I hope you're well. Awesome work on Amethyst, truly!
Is it possible to still be able to focus on floated windows using the shortcut key?
E.g. if I have `Terminal.app` set to float, I can't get access to it via `mod1 + j/k`.

Just wondering if that's expected behaviour?

**Comment by ianyh** [comment]
@maikeldaloo it's the expected behavior, but might not be the right one. I think there's another issue for it already, though.
